.github/workflows/doc.yml

```diff
@@ -0,0 +1,46 @@
+name: "Run: Build documentation"
+
+permissions: {}
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}-doc-${{ github.run_id }}"
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    paths:
+      - 'doc/**'
```

Comment:
Pull requests push to arbitrary branches with repository permissions.
Delete this.

Reply:
ok